misc/identity-keys/src/identity-keys.ts

```diff
@@ -52,6 +53,40 @@ export class IdentityKeys implements IIdentityKeys {
     return this.identityKeys.keys.includes(account);
   }
 
+  public async prepareRegistrationViaRecaps({
+    domain,
+    recapObject
+  }: {
+    domain: string
+    recapObject: {
+      att: Record<string,any>
+    }
+  }) {
+    const encodedRecap = objectToHex(recapObject);
```

Comment:
This should be unpadded base64url not hex. Maybe use `makeBase64UrlSafe()`?

Reply:
I know the naming is off but that's essentially what it is I made sure it produces the same string bartek's implementation did